.github/workflows/publish-react-native.yml

```
@@ -49,6 +49,16 @@ jobs:
         working-directory: libs/sdk-react-native
         run: yarn
 
+      - name: Pack for archival
+        working-directory: libs/sdk-react-native
+        run: yarn pack --out sdk-react-native.tgz
+
+      - name: Archive the package
+        uses: actions/upload-artifact@v3
+        with:
+          name: react-native-${{ inputs.package-version || '0.0.2' }}
+          path: libs/sdk-bindings/sdk-react-native/sdk-react-native.tgz
```

Just a path fix, then looks good

```suggestion
          path: libs/sdk-react-native/sdk-react-native.tgz
```